src/hooks/useFarmStats.js

```diff
@@ -67,10 +73,12 @@ export function useFarmStats(farmAddress) {
           farmDaysLeft = Math.ceil(farmTimeLeft / (1000 * 60 * 60 * 24)); 
         }
         
+        const _totalStaked = JSBI.BigInt(staked);
```

Comment:
Let's not use a `_` prefix—shadowing the above variable is fine, as we don't wanna access it anyway.